src-electron/db/zap-schema.sql

```diff
@@ -186,6 +186,7 @@ CREATE TABLE IF NOT EXISTS "COMMAND_ARG" (
   "TYPE" text,
   "IS_ARRAY" integer,
   "PRESENT_IF" text,
+  "IS_NULLABLE" integer,
```

Comment:
Why integer here, not boolean?

Reply:
There is no BOOLEAN types in SQLITE. Boolean are by convention integers saved as 1 or 0.